pkg/generator/generator.go

```diff
@@ -210,7 +210,14 @@ func (g Generator) GenerateConfig(ctx context.Context, installation, app, ref st
 	name := generateResourceName(app, ref)
 	meta := metav1.ObjectMeta{
 		Name:      name,
-		Namespace: "giantswarm",
+		Namespace: namespace,
+		Labels: map[string]string{
+			"app.kubernetes.io/managed-by": "Helm",
```

Comment:
It would be good to also add `"giantswarm.io/managed-by": project.Name()`

Reply:
Done.